src/core/Akka.API.Tests/CoreAPISpec.ApproveCore.approved.txt

```diff
@@ -4235,39 +4250,52 @@ namespace Akka.Routing
     {
         public SmallestMailboxRoutingLogic() { }
         public SmallestMailboxRoutingLogic(int next) { }
         public override Akka.Routing.Routee Select(object message, Akka.Routing.Routee[] routees) { }
     }
+    public sealed class TailChopping : Akka.Routing.RoutingLogic
+    {
+        public TailChopping(Akka.Actor.IScheduler scheduler, System.TimeSpan within, System.TimeSpan interval) { }
+        public override Akka.Routing.Routee Select(object message, Akka.Routing.Routee[] routees) { }
+    }
     public sealed class TailChoppingGroup : Akka.Routing.Group
     {
         public TailChoppingGroup(Akka.Configuration.Config config) { }
-        public TailChoppingGroup(string[] routeePaths, System.TimeSpan within, System.TimeSpan interval) { }
+        public TailChoppingGroup(System.Collections.Generic.IEnumerable<string> routeePaths, System.TimeSpan within, System.TimeSpan interval) { }
+        public TailChoppingGroup(System.Collections.Generic.IEnumerable<string> routeePaths, System.TimeSpan within, System.TimeSpan interval, string routerDispatcher) { }
+        public System.TimeSpan Interval { get; }
+        public System.TimeSpan Within { get; }
         public override Akka.Routing.Router CreateRouter(Akka.Actor.ActorSystem system) { }
+        public override System.Collections.Generic.IEnumerable<string> GetPaths(Akka.Actor.ActorSystem system) { }
         public override Akka.Util.ISurrogate ToSurrogate(Akka.Actor.ActorSystem system) { }
-        public override Akka.Routing.Group WithDispatcher(string dispatcher) { }
+        public Akka.Routing.TailChoppingGroup WithDispatcher(string dispatcher) { }
         public class TailChoppingGroupSurrogate : Akka.Util.ISurrogate
         {
             public TailChoppingGroupSurrogate() { }
             public System.TimeSpan Interval { get; set; }
-            public string[] Paths { get; set; }
+            public System.Collections.Generic.IEnumerable<string> Paths { get; set; }
+            public string RouterDispatcher { get; set; }
             public System.TimeSpan Within { get; set; }
             public Akka.Util.ISurrogated FromSurrogate(Akka.Actor.ActorSystem system) { }
         }
     }
     public sealed class TailChoppingPool : Akka.Routing.Pool
     {
-        public TailChoppingPool(int nrOfInstances, Akka.Routing.Resizer resizer, Akka.Actor.SupervisorStrategy supervisorStrategy, string routerDispatcher, System.TimeSpan within, System.TimeSpan interval, bool usePoolDispatcher = False) { }
         public TailChoppingPool(Akka.Configuration.Config config) { }
         public TailChoppingPool(int nrOfInstances, System.TimeSpan within, System.TimeSpan interval) { }
+        public TailChoppingPool(int nrOfInstances, Akka.Routing.Resizer resizer, System.TimeSpan within, System.TimeSpan interval, Akka.Actor.SupervisorStrategy supervisorStrategy, string routerDispatcher, bool usePoolDispatcher = False) { }
```

Comment:
Reverting this back to its original signature. Backwards compat trumps 100% alignment with JVM.